### PR TITLE
Move gatk4/svannotate to nf-test

### PR DESCRIPTION
**Pull request by matthdsm**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/gatk4/svannotate/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.annotated_vcf.get(0).get(1)).name,
+                    process.out.index.get(0).get(1),
+                    )
+                .match("test_gatk4_svannotate") }
+            )
+        }
+
```
Comment (maxulysse) [inline modules/nf-core/gatk4/svannotate/tests/main.nf.test:50]:
```suggestion
```

Comment (maxulysse) [reply modules/nf-core/gatk4/svannotate/tests/main.nf.test:50]: I have been chasing these in all the tests I check, is this created by tools?

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/gatk4/svannotate/tests/main.nf.test (hunk 1 above)
Comment (matthdsm) [reply modules/nf-core/gatk4/svannotate/tests/main.nf.test]: Yes!

**Review by nvnieuwk — APPROVED** [review]
LGTM

**Pull request merged**